### DISCUSSION TITLE
What are the alternatives to this library?

### DISCUSSION THREAD

**Discussion by yamiteru** [yamiteru/isitfast#10] [Q&A]
I want to write a brief comparison of `isitfast` and other ~4 libraries but I'm not sure which ones I should choose.

I mean obviously `benchmark.js` has to be one of them. But what should be the remaining ~3 libs?

Any ideas?

**Comment by yamiteru** [comment]
Currently these are the two closest alternatives that I know of:

- [benchmark.js](https://github.com/bestiejs/benchmark.js)
- [tinybench](https://github.com/tinylibs/tinybench)

**Comment by H4ad** [comment]
I already use [cronometro](https://github.com/ShogunPanda/cronometro), has support for ESM.
